### ISSUE TITLE
Enable amounts in URIs

### ISSUE THREAD

**Issue by hank** [Litecoin-Java/bitcoin-wallet#67]
For some reason, URI scanning like the following doesn't seem to work:

```
litecoin:LSomeAddressaa6a6a66a6a6&amount=1.2345
```

**Comment by hank** [comment]
Potentially add currency code option, like:

```
litecoin:LSomeAddressaa6a6a66a6a6&amount=1.23&currency=USD
```

Which would convert to LTC using the rate of LTC/USD in the app.

**Comment by Vanhouttethomas** [comment]
How would that rate be calculated? Using the API of BTC-e or another exchange?

**Comment by hank** [comment]
@Vanhoutte right, it would use whatever the app has decided is the correct exchange to use, which would mean the user would have to confirm the number of LTC with the merchant before sending (seems reasonable to me).  It sucks that this is the case, but I don't see a good way around it.
